cmd/internal-ip/main.go

```diff
@@ -1,63 +1,87 @@
 package main
 
 import (
+	"errors"
+	"flag"
 	"fmt"
 	"net"
 	"os"
 	"strings"
 )
 
+// Set display texts to defaults.
+var fullText = "unknown"
+var shortText = "unknown"
+
 func main() {
+	var iface = &net.Interface{}
+
+	// Check if interface parameter was passed in
+	var ifaceNameFlag = flag.String("ifaceName", "", "-ifaceName wlp2s0")
+	flag.Parse()
+
+	if ifaceNameFlag == nil || *ifaceNameFlag == "" {
```

Comment:
```suggestion
	if *ifaceNameFlag == "" {
```

Reply:
It suffices to check the dereferenced flag value to be equal to the default value (```""``` for strings) in order to determine whether the flag was set or not. ```nil``` check is a good thought, but unnecessary in this case.